src/rumble.c

```diff
@@ -3,8 +3,8 @@
 
 // TODO: replace nustuff with defines
 
-u16 D_80074260 = 0;
-s32 D_80074264 = 0;
+u16 RumbleMaxDuration = 0;
+s32 RumbleButtons = 0;
```

Comment:
These should be lowerCamelCase, and possibly prefixed like `rumble_maxDuration` and `rumble_buttons`

Reply:
@ethteck do you think we should prefix it like rumble_ or just rumbleMaxDuration